### PR TITLE
Update docs for release v0.1.5

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 1]
```diff
@@ -70,13 +70,20 @@ To use a secret in an environment variable in a Pod, follow these steps.
...
+| `tlsSecret` | The TLS secret contains the `use_tls`, `tls_allow_insecure`, `hostname_verification_enabled`, `tls_trust_cert_path` fields.|
+
 ## Packages
 
 Function Mesh supports running Pulsar connectors in Java.
 
 | Field | Description |
 | --- | --- |
 | `jarLocation` | Path to the JAR file for the connector.|
+| `extraDependenciesDir` | It specifies one or more dependent directories for the JAR package. |
```
Comment (nlu90) [inline docs/connectors/io-crd-config/sink-crd-config.md:86]: only one dir can be specified.

multiple lib/jar can be in that dir.

**Review by nlu90 — CHANGES_REQUESTED** [review]

File: docs/connectors/io-crd-config/sink-crd-config.md (hunk 1 above)
Comment (Huanli-Meng) [reply docs/connectors/io-crd-config/sink-crd-config.md]: Thanks for your comments. PTAL. thanks.

**Review by tuteng — APPROVED** [review]

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 2]
```diff
@@ -70,13 +70,20 @@ To use a secret in an environment variable in a Pod, follow these steps.
 2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
 3. Modify your image and or command line so that the program looks for values in the specified environment variables.
 
+Pulsar clusters supports using TLS or other authentication plugin for authentication.
+
+| Field | Description |
+| --- | --- |The authentication secret contains the `clientAuthenticationPlugin` and `clientAuthenticationParametersPath` fields.|
+| `tlsSecret` | The TLS secret contains the `use_tls`, `tls_allow_insecure`, `hostname_verification_enabled`, `tls_trust_cert_path` fields.|
+
```
Comment (Jennifer88huang) [inline docs/connectors/io-crd-config/sink-crd-config.md:78]: It seems that there is sth wrong with the table format

Comment (freeznet) [reply docs/connectors/io-crd-config/sink-crd-config.md:78]: I suggest to split this part into two sub sections.

- TLS Secret
| Field | Description |
| --- | --- |
| `tlsAllowInsecureConnection` | Allow insecure tls connection. |
| `tlsHostnameVerificationEnable` | Enable hostname verification. |
| `tlsTrustCertsFilePath` | tls trust cert file path. |

- Auth Secret
| Field | Description |
| --- | --- |
| `clientAuthenticationPlugin` | Client authentication plugin. |
| `clientAuthenticationParameters` | Client authentication params. |

**Review by freeznet — COMMENTED** [review]
PTAL of my comment, thanks.

**Comment by Huanli-Meng** [comment]
@freeznet Thanks for your comments. Doc has been updated. PTAL again. Thanks.

**Review by freeznet — APPROVED** [review]
LGTM

**Comment by Huanli-Meng** [comment]
@nlu90 Please help double review on this PR. Feel free to approve it if there is no more comment. Thanks

**Review by nlu90 — APPROVED** [review]

**Pull request merged**